src/index.tsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
+import { registerFetch, registerLogger } from 'conseiljs';
+import fetch from 'node-fetch';
+import * as log from 'loglevel';
+
 import Root from './containers/Root';
 import { configureStore, history } from './store/configureStore';
 import './utils/i18n';
 import './styles/app.global.scss';
 
 const store = configureStore();
 
+const logger = log.getLogger('conseiljs');
+logger.setLevel('debug', false);
```

Comment:
In an ideal world, this would be a configuration item.

Reply:
fixed